website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -7,36 +7,34 @@ description: >
 ---
 
 Karpenter sets a Kubernetes [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) on each node it provisions.
-The finalizer specifies additional actions the Karpenter controller will take in response to a node deletion request.
-These include:
-
-* Marking the node as unschedulable, so no further pods can be scheduled there.
-* Evicting all pods other than daemonsets from the node.
-* Terminating the instance from the cloud provider.
-* Deleting the node from the Kubernetes cluster.
+The finalizer signals to the Karpenter Termination controller to gracefully shutdown a node that receives a deletion request. Karpenter deprovisioning can be executed by Karpenter through the Deprovisioning Controler or by the user through manual deprovisioning.
+
+### Deprovisioning Controller
+Karpenter automatically discovers deprovisionable nodes and spins up replacements when needed. Karpenter deprovisions nodes by executing one [automatic method](#methods) at a time, in order of Expiration, Drift, Emptiness, then Consolidation. Each method varies slightly but they all follow the standard deprovisioning process:
+1. Generate a list of deprovisionable nodes and sort them as needed.
+   1. If there are no deprovisionable nodes, continue to the next deprovisioning method.
+2. For each deprovisionable node, execute a scheduling simulation with the pods on the node to find if any replacement nodes are needed.
+3. Cordon to-be-deprovisioned node(s) so no further pods can be scheduled on the node.
+4. Pre-spin any replacement nodes needed as calculated in Step (2), and wait for them to become ready.
+   1. If a replacement node fails to initialize, un-cordon the to-be-deprovisioned node(s), and restart from at Step (1).
+5. Delete the to-be-deprovisioned node(s) and wait for the termination controller to gracefully shutdown the node(s).
+6. Once the termination controller terminates the node, go back to Step (1), starting at the the first method again.
+
+### Termination Controller
+When a Karpenter node is deleted, the Karpenter finalizer will block deletion and the APIServer will set the `DeletionTimestamp` on the node, allowing Karpenter to gracefully shutdown the node, modeled after [K8s Graceful Node Shutdown](https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown). Karpenter's graceful shutdown process will:
```

Comment:
In some cases you squish `APIServer` together and other place it has a space. It doesn't matter to me either way but can we be consistent throughout the docs?